src/renderer.js

```diff
@@ -219,68 +214,35 @@ const iconv = require('iconv-lite');
         throw new Error('Not able to get key from profile directory.');
     }
 
-    function decrypt(data, iv, key) {
-        const decipher = forge.cipher.createDecipher('3DES-CBC', key);
+    // Adapted from https://github.com/lclevy/firepwd/blob/master/firepwd.py
+    function decryptPbe(decodedItemSeq, password, globalSalt) {
+        const data = decodedItemSeq[1].value;
+        const pbkdf2Seq = decodedItemSeq[0].value[1].value[0].value[1].value;
+        const salt = pbkdf2Seq[0].value;
+        const iterations = pbkdf2Seq[1].value.charCodeAt();
+        const keyLength = pbkdf2Seq[2].value;
+        // Prepending 0x040e, where 0x04 = octetstring, 0x0e = string length (14)
+        const iv = '' + decodedItemSeq[0].value[1].value[1].value[1].value;
```

Comment:
Looks like a bug on Github's end. Copy this into some text and you should see that these are characters that don't have a Unicode representation.